tests/perf-regression-mv.100threads.30M-keys-i3.yaml

```diff
@@ -0,0 +1,72 @@
+test_duration: 70
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop seq=1..30000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop seq=1..30000000"
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=50m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=50m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -errors ignore -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+user_prefix: 'perf-regression-VERSION'
+failure_post_behavior: destroy
+space_node_threshold: 644245094
+ami_id_db_scylla_desc: 'VERSION_DESC'
+ip_ssh_connections: 'private'
+instance_provision: 'spot_low_price'
+experimental: 'true'
```

Comment:
do we need this line in performance testing?

Reply:
Yes. It must for materialized view test